CHANGELOG.md

```diff
@@ -16,6 +16,10 @@ You may also find the [Upgrade Guide](https://rust-random.github.io/book/update.
 - Unit tests for value stability of distributions added (#888)
 
 ## [0.7.1] - 2019-09-13
+### Yanked
+This release was yanked since it depends on `rand_core::OsRng` added in 0.5.1,
+yet only specifies a dependency on version 0.5.0 (#890).
```

Comment:
Maybe add a sentence that this breaks semver.

Reply:
It's not incorrect versioning of the crate itself, it's an incorrect dependency specification. I think it's just confusing to call that a semver issue.

Reply:
Fair enough, but I still think we should mention the consequence (broken builds).